session/tests/SessionCodecsTests.cpp

```
@@ -0,0 +1,188 @@
+		if (!encoded) CHECK(encoded.error().code == CodecErrorCode::FingerprintMismatch);
+	}
+}
+
+namespace configured_synth_tests {
+	int failures = 0;
+	void run();
+}
+
+int main() {
+	configured_synth_tests::run();
+	validManifestRoundTripsSemantically();
```

Cppcheck (2.21.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)